app/views/staff/uploads/actual_histories/_errors_table.html.haml

```diff
@@ -0,0 +1,20 @@
+%p.govuk-body
+  Below are the errors in your actual data upload, fix the errors listed
+  and try again.
+.govuk-inset-text
+  No data has been uploaded.
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-table__caption--m Upload errors
+  %thead.govuk-table__head
+    %tr.govuk-table__row
+      %th.govuk-table__header{scope: "col"} Column
+      %th.govuk-table__header{scope: "col"} Row
+      %th.govuk-table__header{scope: "col"} Current Value
+      %th.govuk-table__header{scope: "col"} Error description
+  %tbody.govuk-table__body
+    - @errors.each do |error|
+      %tr.govuk-table__row
+        %td.govuk-table__cell= error.column
+        %td.govuk-table__cell= error.row_number
+        %td.govuk-table__cell= error.value
+        %td.govuk-table__cell= error.message
```

Comment:
It's neat now the details of the error information are described here. But possibly it's also good to have them documented in the "has a helpful error message" example of the `ActualHistorySpec`?

Reply:
I think this is the same comment as the one above, I've added expectations for eaach attribute of the error now as suggested. 👍